### DISCUSSION TITLE
jails comes to Truenas SCALE!!!

### DISCUSSION THREAD

**Discussion by qudiqudi** [Jip-Hop/jailmaker#112] [Show and tell]
Thanks to @Jip-Hop and all the contributors effort, the idea behind jailmaker is now considered to be baked into Truenas SCALE.
https://www.truenas.com/docs/scale/scaletutorials/apps/sandboxes/
The article is even mentioning this github project.
WELL DONE EVERYONE!!! Gotta love the open source community! ❤️

**Comment by Jip-Hop** [comment]
While I wouldn't go so far as to say it's baked into TrueNAS SCALE yet, it's definitely exciting. For now I will continue to work on jailmaker as mentioned in the [congratulation to Sandboxes (Jail-like Containers) // jailmaker soon obsolete](https://github.com/Jip-Hop/jailmaker/discussions/79) discussion.
